integration-cli/docker_cli_build_test.go

```diff
@@ -3753,6 +3756,22 @@ ENV    abc 'yyy'
 RUN    [ $abc = \'yyy\' ]
 ENV    abc=
 RUN    [ "$abc" = "" ]
+ENV    abc=\$foo
+RUN    [ "$abc" = "\$foo" ]
```

Comment:
@duglin Could be dangerous, if the builder substitutes both `\$foo` in RUN and in ENV. Then you have double negative yielding a positive result :(